### PR TITLE
support log ttl and add test

### PR DESCRIPTION
**Pull request by siddontang**
@BusyJay @hhkbp2 @zhangjinpeng1987

### CODE REVIEW

File: tests/test_rocksdb_options.rs [hunk 1]
```diff
@@ -139,11 +141,22 @@ fn test_create_info_log() {
...
     let info_dir = TempDir::new("_rust_rocksdb_test_info_log_dir").expect("");
     opts.create_info_log(info_dir.path().to_str().unwrap()).unwrap();
+
     let db = DB::open(opts, path.path().to_str().unwrap()).unwrap();
 
     assert!(Path::new(info_dir.path().join("LOG").to_str().unwrap()).is_file());
 
+    thread::sleep(Duration::from_secs(2));
+
+    // The LOG must be rolled many times.
```
Comment (hhkbp2) [inline tests/test_rocksdb_options.rs]: Any way to test log file size?

File: tests/test_rocksdb_options.rs [hunk 2]
```diff
@@ -139,11 +141,22 @@ fn test_create_info_log() {
...
+    let count = info_dir.path().read_dir().unwrap().count();
+    assert!(count > 1);
+
     drop(db);
+
```
Comment (hhkbp2) [inline tests/test_rocksdb_options.rs]: Remove empty line.

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**